timescaledb/how-to-guides/install-timescaledb/installation-docker.md

```diff
@@ -1,94 +1,188 @@
-## Docker Hub [](docker)
+# Install self-hosted TimescaleDB from a pre-built container
+You can install a self-hosted TimescaleDB instance on any local system, from a
+pre-built container. This is the simplest method to install a self-hosted
+TimescaleDB instance, and it means you always have access to the latest version
+without worrying about local dependencies. You can access the Docker image
+directly from locally installed PostgreSQL client tools such as `psql`.
 
-#### Quick start
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than the pre-built
+container provided here, you could encounter errors following these
+instructions. It is safest to remove any existing PostgreSQL installations
+before you begin. If you want to keep your current PostgreSQL installation, do
+not install TimescaleDB using this method.
+TimescaleDB using this method.
+[Install from source](/how-to-guides/install-timescaledb/installation-source/)
+instead.
+</highlight>
+
+<procedure>
 
-Start a TimescaleDB instance, pulling our Docker image from [Docker Hub][] if it has not been already installed:
+### Installing self-hosted TimescaleDB from a Docker container
+1.  Install Docker, if you don't already have it. For packages and
+    instructions, see the [Docker installation documentation][docker-install].
+1.  At the command prompt, run the TimescaleDB Docker image:
+    ```bash
+    docker pull timescale/timescaledb:latest-pg14
```

Comment:
we do have more docker images, the most important one being timescaledb-ha which is more full featured as is has more extensions pre-installed also it does not share some of the limitations regarding collations that the alpine-based one has

Reply:
I don't think we need to mention that here, wouldn't it be better to mention it in the HA docs? (we can always include a pointer back to this page for instructions, though).

Reply:
I really want someone to clarify this from engineering and (potentially) see if we can get a name change. Nearly every answer we give in community Slack right now is to use the `timescaledb-ha` image because it has things like the Toolkit pre-installed (among other things). But to a tee, almost everyone is confused why we're suggesting an "HA" image for their install/testing.

So, totally agree with @svenklemm that someone needs to decide this. Because it looks really weird that we pretty much tell everyone to use one image but don't mention it in docs anywhere. 🤷‍♂️